Bottom: 1b902efa3a58d7d0d83ed241c390dad28dcd55a1
Top:    c36ce8e2bad54aa2dec9f15b2e03af292038b928
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 15:46:20 +0100

Refresh of crasy-test

---

diff --git a/src/draw_funcs.c b/src/draw_funcs.c
index d6e285e..0bdd30f 100644
--- a/src/draw_funcs.c
+++ b/src/draw_funcs.c
@@ -81,7 +81,7 @@ draw_pad_mask (PadType *pad, void *userdata)
   if (pad->Mask <= 0)
     return;
 
-  _draw_pad (Output.fgGC, pad, true, true);
+  _draw_pad (Output.pmGC, pad, true, true);
 }
 
 static void
